### PR TITLE
Make updates to functions to accommodate both pediatrics and adults notebooks

### PR DESCRIPTION
**Pull request by andersonmeredithk**

### CODE REVIEW

File: GrowthViz-adults.py [hunk 1]
```diff
@@ -262,65 +259,50 @@ def handle_selection_change(_event, _widget):
...
 obs_wbmi[obs_wbmi['subjid'] == 'd88d3987-93ff-0820-286f-754cd971012d'] # b5a84a9d-dd7c-95cb-5fd9-3c581a72c812, 867a461b-7cb8-76aa-9891-42369a9899e8 is an example with the underweight line
 
 
-# In[22]:
+# In[21]:
 
 
 # display all charts at the same time
-charts.overlap_view_all(obs_df=obs_wbmi, id=val, param='WEIGHTKG', include_carry_forward=True, 
+charts.overlap_view_adults(obs_df=obs_wbmi, subjid=val, param='WEIGHTKG', include_carry_forward=True,
```
Comment (dchud) [inline GrowthViz-adults.py:287]: Good to use `subjid` here instead of `id`, which is another python built-in!

**Review by dchud — CHANGES_REQUESTED** [review]
Minor details noted in prior comments.

File: sumstats.py [hunk 2]
```diff
@@ -5,26 +5,32 @@
...
+  # create mean/sd values to merge to adult data for z-score calculations
+  dta_forz_long = percentiles_clean[['Mean', 'Sex', 'param', 'age', 'sd']]
   def label_param (row):
     if row['param'] == 'WEIGHTKG' : return 'weight'
     if row['param'] == 'BMI' : return 'bmi'
     if row['param'] == 'HEIGHTCM' : return 'height'
   param_col = dta_forz_long.apply(lambda row: label_param(row), axis=1)
   dta_forz_long = dta_forz_long.assign(param2=param_col.values)
-  dta_forz = dta_forz_long.pivot_table(index=['sex', 'age_years'], columns='param2', values=['Mean', 'sd'], aggfunc='first') #.reset_index()
+  dta_forz = dta_forz_long.pivot_table(index=['Sex', 'age'], columns='param2', values=['Mean', 'sd'], aggfunc='first') #.reset_index()
```
[the remark this answers is not in the document]
Comment (andersonmeredithk) [reply sumstats.py:17]: The capitalization is to make adults variable names consistent with pediatrics percentiles data so certain functions work for both notebooks - is that a reasonable approach?

File: charts.py [hunk 3]
```diff
@@ -76,58 +80,51 @@ def overlap_view(obs_df, subjid, param, include_carry_forward, include_percentil
...
   selected_param_plot.legend(loc="upper left", bbox_to_anchor=(1.05, 1))
-
+  plt.title(param)
   return selected_param_plot
 
-def overlap_view_all(obs_df, id, param, include_carry_forward, include_percentiles, wt_df, bmi_df, ht_df):
+def overlap_view_pediatrics(obs_df, subjid, param, include_carry_forward, include_percentiles, wt_df, ht_df):
   """
-  Creates a version of overlap_view() that display three charts (one each for wt, bmi, and ht) together
+  Creates a chart showing the trajectory for an individual with all values present. All values will
```
[the remark this answers is not in the document]
Comment (andersonmeredithk) [reply charts.py:88]: I am not following a particular style guide, although I can if you direct me to one you recommend. I added a separate ticket for this, thanks.

File: sumstats.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (dchud) [reply sumstats.py]: It is reasonable! Perhaps just add a comment right above it that says something to that effect?

File: charts.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (dchud) [reply charts.py:88]: Great! I added a comment with some suggestions over on #18.

No need to do it all now for this PR!

File: processdata.py [hunk 4]
```diff
@@ -5,103 +5,127 @@
...
+  df = obs_df
+  if mode == 'adults':
+    df.rename(columns={'result':'clean_value', 'age_years':'age'}, inplace=True)
+    #obs_df['age'] = obs_df['age_years'] 
+  elif mode == 'pediatrics':  
+    df['age'] = df['agedays'] / 365 
+    df.drop(columns=['agedays'], inplace=True)
+  df['clean_cat'] = df['clean_value'].astype('category')
+  df['include'] = df.clean_value.eq("Include")
+  return df
```
[the remark this answers is not in the document]
Comment (dchud) [reply processdata.py:19]: Just a minor change here, I would follow the advice at https://stackoverflow.com/a/256260 with something like:

```python
else:
  raise ValueError("Mode must 'adults' or 'pediatrics'")
```

Just to be super-explicit that this is an error. `ValueError` is a built-in Exception.

**Review by dchud — CHANGES_REQUESTED** [review]
Just the one `ValueError` recommendation.

File: processdata.py [hunk 5]
```diff
@@ -5,103 +5,127 @@
...
-    dta[col] = np.where((dta['decade'] == 1) & (dta['age_years'] < 110), (dta[col] + dta[col].shift(1))/2, dta[col])
+    dta[col] = np.where((dta['decade'] == 1) & (dta['age'] < 110), (dta[col] + dta[col].shift(1))/2, dta[col])
   dta.drop(columns={'decade'}, inplace=True)
-  col_list = ['param', 'sex', 'age_years'] + mcol_list
+  col_list = ['param', 'Sex', 'age'] + mcol_list
   dta = dta.reindex(columns=col_list)
   return dta
 
-def keep_age_range(df):
+# figure out issue with duplicated header row
```
[the remark this answers is not in the document]
Comment (dchud) [reply processdata.py]: Looks like you addressed this in the CSV file? Resolving, thinking "yes".

**Pull request merged**